download/download.adoc

```diff
@@ -1,133 +0,0 @@
-= Download
-:awestruct-description: Download the community release of OptaPlanner, including examples, sources and documentation.
-:awestruct-layout: normalBase
-:awestruct-priority: 1.0
```

Comment:
you also lost this, restoring that.
That affects sitemap.xml which affects how important google thinks this page is